src/files-and-videos/files-page/FilesPage.test.jsx

```diff
@@ -204,10 +213,10 @@ describe('FilesAndUploads', () => {
           await waitFor(() => {
             addFilesButton = screen.getByLabelText('file-input');
           });
-          await act(async () => {
+          await waitFor(() => {
             userEvent.upload(addFilesButton, file);
-            await executeThunk(validateAssetFiles(courseId, [file]), store.dispatch);
           });
```

Comment:
I don't think you need a `waitFor` here. A `waitFor` will execute the condition inside it multiple times till it stops throwing an error. So here it will just run it once.

Reply:
Cool thanks, I will fix these.

I actually don't even understand why we have to run the thunks explicitly here.. shouldn't they be triggered by the user event?

Reply:
I'm not sure either, without looking deeper into the code it's hard to know. However, sometimes when I've followed this pattern it is to set up everything before the UI interactions begin in the tests.